components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -101,9 +102,7 @@ class PlacesApi(path: String) : PlacesManager, AutoCloseable {
         if (handle != 0L) {
             if (writeHandle != 0L) {
                 try {
-                    rustCall(this) { err ->
-                        LibPlacesFFI.INSTANCE.places_api_return_write_conn(handle, writeHandle, err)
-                    }
+                    this.api.apiReturnWriteConn()
```

Comment:
I don't think this is correct - it should remain using the old technique while we still have that duality of both handle and uniffi connections. We will need *something* to kill the uniffi side of the world, but that might just be able to be a `this.writeConn.close()` method.

(To be clear, this probably isn't going to hurt because we will kill the old handle stuff before merging to main, but it still makes sense to me to keep things clear)

Reply:
hmmm this is an interesting point. So it actually makes sense to simply not uniffi these connection-level handle problems and mainly just focus on being able to call interrupt on the new connections.